components/Map/style.css

```diff
@@ -0,0 +1,13 @@
+.custom-marker-cluster {
```

Comment:
Move this to `styles/global.css` and you won't need any extra dependencies :pray:

Reply:
Noted.

Reply:
Remove the external depdency, please :pray:

Reply:
Ohh I got it now, we don't need next-global-css. It's literally in the name, global 😅 anyway, I'm on it :)

Reply:
> Ohh I got it now, we don't need next-global-css. It's literally in the name, global sweat_smile anyway, I'm on it :)

Update:

![image](https://user-images.githubusercontent.com/80540164/181629920-d90226b0-96f2-41d5-bdd2-55be1753c397.png)

@nelsonmestevao

Reply:
I guess we will to use global css until the new release happens :'(  (looking forward to https://github.com/yuzhva/react-leaflet-markercluster/pull/194)

Reply:
Ahaha true, last commit was two years ago :')